lib/mayaUsd/python/wrapPrimUpdaterManager.cpp

```diff
@@ -128,5 +133,6 @@ void wrapPrimUpdaterManager()
         .def("canEditAsMaya", canEditAsMaya)
         .def("discardEdits", discardEdits)
         .def("duplicate", duplicate, duplicate_overloads())
+        .def("isEditedAsMayaOrphaned", isEditedPrimOrphaned)
```

Comment:
Why does the python wrapper have a different name than the internal helper function?

Reply:
The Python function has the same name as the real C++ function. The gave the internal function a different name because otherwise it would confuse the compiler (and code reader) which would see the same name as overload. The real C++ function is named isEditedAsMayaOrphaned like the Python function.